doc/classes/String.xml

```diff
@@ -1072,6 +1072,7 @@
 			<return type="String" />
 			<param index="0" name="right" type="StringName" />
 			<description>
+				Appends [param right] at the end of this [String], returning a [String].
```

Comment:
This is the only description out of 4 not to mention that this is also known as string concatenation 😛